modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/dynamic_include/bottom.jsp

```diff
@@ -44,48 +44,36 @@ sharingURL.setWindowState(LiferayWindowState.POP_UP);
 		});
 	}
 
-	var Sharing = {};
+	Liferay.Sharing = Liferay.Sharing || {};
```

Comment:
Based on previous code, it's unlikely `Sharing` is coming from anywhere else, since it was always wiped out with `var Sharing = {}`.

Not sure about other reviews, but why not just do this?

```javascript
Liferay.Sharing = {
    manageCollaborators: function() {...},
    share: function() {...},
}
```

Reply:
@jbalsas I know what you mean, and that's what I first did ... but I changed it based on what we had said in previous discussions. I basically remember what @wincent commented [here](https://github.com/wincent/liferay-portal/pull/327#discussion_r437362199) and wanted to use the same approach (although I agree that in this case it's simple enough to make the exception)

Reply:
Well... in there it says:

> In other words, we have two places that in an order-dependent way will produce a different globally-visible result or other. Specifically, the last-to-write clobbers what was there before.
>
> To be robust, every place that wants to define Liferay.Users should check to see if it already exists (that way, if execution order ever changes, it will still work).

I don't think it's the same case since you **DO NOT** have 2 places where you assign stuff to the `Sharing` object. This is no exception to what @wincent told you there afaict.

If anything, he was warning you about doing `Liferay.Foo = Bar` in different places since assignment is a destroying action.

Reply:
Yes, I know here we just have one place in which `Sharing` will be declared and assigned to `Liferay.Sharing`, but I just wanted to confirm, so thanks